src/lib.rs

```diff
@@ -76,21 +74,19 @@
 //! # extern crate rand;
 //! #
 //! # fn main() {
-//! # use x25519_dalek::generate_secret;
-//! # use x25519_dalek::generate_public;
+//! # use x25519_dalek::Ephemeral;
 //! # use rand::thread_rng;
 //! #
 //! # let mut alice_csprng = thread_rng();
-//! # let     alice_secret = generate_secret(&mut alice_csprng);
-//! # let     alice_public = generate_public(&alice_secret);
+//! # let     alice_secret = Ephemeral::generate_secret(&mut alice_csprng);
+//! # let     alice_public = Ephemeral::generate_public(&alice_secret);
 //! #
 //! # let mut bob_csprng = thread_rng();
-//! # let     bob_secret = generate_secret(&mut bob_csprng);
-//! # let     bob_public = generate_public(&bob_secret);
+//! # let     bob_secret = Ephemeral::generate_secret(&mut bob_csprng);
+//! # let     bob_public = Ephemeral::generate_public(&bob_secret);
 //! #
-//! use x25519_dalek::diffie_hellman;
 //!
```

Comment:
```suggestion
//! #
```
will hide a blank line in rendered docs